### PR TITLE
change: Add support for Accept headers with multiple MIME types

### PR DESCRIPTION
**Pull request by bveeramani**
*Issue #, if available:*

*Description of changes:*
* Changed implementation of `default_output_fn`
* Added test to `test_default_inference_handler`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker_mxnet_serving_container/default_inference_handler.py [hunk 1]
```diff
@@ -108,10 +118,10 @@ def default_output_fn(self, prediction, accept):
             sagemaker_inference.errors.UnsupportedFormatError: if an unsupported content type is used.
 
         """
-        if accept in self.VALID_CONTENT_TYPES:
-            return encoder.encode(prediction.asnumpy().tolist(), accept)
-        else:
-            raise errors.UnsupportedFormatError(accept)
+        for content_type in parse_accept(accept):
+            if content_type in self.VALID_CONTENT_TYPES:
```
Comment (metrizable) [inline src/sagemaker_mxnet_serving_container/default_inference_handler.py:122]: could use `if any(..)`

File: src/sagemaker_mxnet_serving_container/utils.py [hunk 2]
```diff
@@ -58,3 +58,18 @@ def read_data_shapes(path, preferred_batch_size=1):
...
     return data_names, data_shapes
+
+
+# TODO (@bvveeram): This function is also implemented in the
+# sagemaker-inference package. Once the MXNet 1.4 EIA image is deprecated, pin
+# the sagemaker-inference dependency to >= 1.5.0, remove this function, and use
+# the equivalent function in sagemaker-inference.
+def parse_accept(accept):
+    """Parses the Accept header sent with a request.
+    Args:
```
Comment (metrizable) [inline src/sagemaker_mxnet_serving_container/utils.py:69]:
```suggestion
    """Parses the Accept header sent with a request.

    Args:
```

File: src/sagemaker_mxnet_serving_container/utils.py [hunk 3]
```diff
@@ -58,3 +58,18 @@ def read_data_shapes(path, preferred_batch_size=1):
         data_shapes.append((name, shape))
 
     return data_names, data_shapes
+
+
+# TODO (@bvveeram): This function is also implemented in the
+# sagemaker-inference package. Once the MXNet 1.4 EIA image is deprecated, pin
+# the sagemaker-inference dependency to >= 1.5.0, remove this function, and use
+# the equivalent function in sagemaker-inference.
```
Comment (metrizable) [inline src/sagemaker_mxnet_serving_container/utils.py:66]: Would you be able to add an issue to the repo after this merge? Thanks!

Comment (bveeramani) [reply src/sagemaker_mxnet_serving_container/utils.py:66]: Yes, will do.

**Pull request merged**